### ISSUE TITLE
Feature Request: Bring Back Custom Status & Do Not Disturb from version 1.10.12

### ISSUE THREAD

**Issue by autoteelar** [SchildiChat/schildichat-desktop#154]
for some reason, turt2live decided to remove some of element's best features simply because "nobody uses them", meanwhile when me and my friends found out these lab features existed we were really happy and excited and used them all the time. just because turt2live doesn't use them and may think for any reason that others don't use them, doesn't mean other people don't use them. i think that decision of his was pre mature.

anyways, if you use the version before this happened (1.10.12 or lower) do not disturb and custom status still work perfectly flawlessly. 

so basically, im requesting if schildichat could keep these features, as i think they are extremely useful, fun, and cool.

and if you go to any other chat platform, say discord for example, so many people use custom status and do not disturb there too, they're pretty popular things in general. 

as for the integration managers, i do not know much about them or used them, if anyone wants them back that knows about them, please say so in the comments.

source post: https://github.com/matrix-org/matrix-react-sdk/pull/8521

**Comment by su-ex** [comment]
Maintenance burden as mentioned on his rationale doesn't sound good. I'm already on my limit and every deviation from Element adds on top.
I might revert the removal of dnd as it looks like a small change, but I'll probably remove it again once it stops working or conflicts a lot while merging.
Also, I won't revert the removal of custom status since it is highly spec-dependant to make a proper implementation.

**Comment by autoteelar** [comment]
okay, well thanks for doing everything you can within reason :D much appreciated

**Comment by autoteelar** [comment]
what does spec-dependant mean? spec? like computer specifications?

**Comment by su-ex** [comment]
> what does spec-dependant mean? spec? like computer specifications?

Status is something which should find its way into the **Matrix** specification so all clients have means to implement this in a proper way working across all of them.

**Comment by autoteelar** [comment]
yeah true, thanks for sharing. they were not implemented the best for sure,
and they made it so only people in your dms could see it, i think a
global/private/friends toggle would be nice for those who want a wider
scope/smaller scope. hopefully the feature comes back better than ever :D

**Comment by darkralts** [comment]
i decided it would be better to lean this onto element, so i made a request over there. some mod or something immediately moved it from feature requests to discussions though without any 2nd thought or explanation though, so i have no idea what the future holds for this specific feature. if anything i might just make my own implementation one way or another.
